src/renderer/dx/DxRenderer.cpp

```diff
@@ -478,6 +479,29 @@ void DxEngine::_ComputePixelShaderSettings() noexcept
     }
 }
 
+// Method Description:
+// - Use DCompositionCreateSurfaceHandle to create a swapchain handle. This API
+//   is only present in Windows 8.1+, so we need to delay-load it to make sure
+//   we can still load on Windows 7.
+// - We can't actually hit this on Windows 7, because only the WPF control uses
+//   us on Windows 7, and they're using the ForHwnd path, which doesn't hit this
+//   at all.
+// Arguments:
+// - <none>
+// Return Value:
+// - An HRESULT for failing to load dcomp.dll, or failing to find the API, or an
+//   actual failure from the API itself.
+[[nodiscard]] HRESULT DxEngine::_CreateSurfaceHandle() noexcept
+{
+    wil::unique_hmodule hDComp{ LoadLibraryEx(L"Dcomp.dll", nullptr, LOAD_LIBRARY_SEARCH_SYSTEM32) };

```

Comment:
The library is technically called `dcomp.dll`. I know that Microsoft's prior file systems are all case insensitive ~unfortunately~, but... you know... It might be a good idea to fix the name.

Assuming that this function is only called from the main thread you can also write:
```cpp
[[nodiscard]] HRESULT DxEngine::_CreateSurfaceHandle() noexcept
{
    static decltype(&DCompositionCreateSurfaceHandle) fn;

    if (!fn)
    {
        wil::unique_hmodule dcomp{ LoadLibraryExW(L"dcomp.dll", nullptr, LOAD_LIBRARY_SEARCH_SYSTEM32) };
        fn = GetProcAddressByFunctionDeclaration(dcomp.get(), DCompositionCreateSurfaceHandle);
        RETURN_LAST_ERROR_IF(fn == nullptr);
    }

    return fn(GENERIC_ALL, nullptr, &_swapChainHandle);
}
```